flixel/graphics/tile/FlxDrawTrianglesItem.hx

```diff
@@ -101,10 +101,17 @@ class FlxDrawTrianglesItem extends FlxDrawBaseItem<FlxDrawTrianglesItem>
 	override public function reset():Void
 	{
 		super.reset();
+		#if (flash || openfl >= "4.0.0")
```

Comment:
we don't support openfl versions older than 9.3.2, we plan to remove the other checks like this, soon. So let's not add more

Reply:
should i remove the check then?

Reply:
yes, please

Reply:
there we go